packages/blade/src/components/Input/BaseInput/StyledBaseInput.web.tsx

```diff
@@ -1,5 +1,6 @@
+/* eslint-disable react/display-name */
```

Comment:
We can add a displayName I think

Reply:
fixed in 717f7a1da44ddbaf76edc37e72d60e182d95576b